scripts/ham-menu.js

```diff
@@ -0,0 +1,14 @@
+const hamburger = document.querySelector('.ham-icon');
+const navMenu = document.querySelector('.nav-menu');
+const nav = document.querySelectorAll('.nav-item');
+
```

Comment:
- [ ] You did an amazing job with the functionality of the mobile menu. Let's work on some minor details to match the Figma design. 🤝️

  - Kindly use the correct color for the menu items. I believe it is `#FFF5E1`
  - Please use the correct color for the separators between the lines. `#6F6C6B`
  - Let's also use the correct color for the close button. `#979493`

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://github.com/Xecyc/My-Portfolio-Project/assets/17243325/e12bde0d-a045-4f88-a8cf-6651a39cc251" width="300">|<img src="https://github.com/Xecyc/My-Portfolio-Project/assets/17243325/f096b604-370b-4ecd-becf-d7a6b317f381" width="300">|